bresenham_numpy.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+from typing import Tuple, Union
+
+
+def bresenham(
+        x0: Union[float, int],
+        y0: Union[float, int],
+        x1: Union[float, int],
+        y1: Union[float, int]
+) -> Tuple[np.ndarray, np.ndarray]:
+    """Return integer coordinates on the line from (x0, y0) to (x1, y1).
+
+    The result will contain both the start and the end point.
+
+    :param x0: The X coordinate of the first point
+    :param y0: The Y coordinate of the first point
+    :param x1: The X coordinate of the second point
+    :param y1: The Y coordinate of the second point
+    :return: The X and Y integer coordinates on the line from (x0, y0) to (x1, y1)
```

Comment:
This should mention the return type (tuple of Numpy arrays).

Reply:
Done!